samples/packages/cdktf-typescript/src/constructs/services.ts

```diff
@@ -1,18 +1,18 @@
 import { Construct } from "constructs";
 import { PackageInfraConfig, PackageServiceDoAppConfig, } from "../package-config";
-import { DigitaloceanProvider } from '@cdktf/provider-digitalocean/lib/provider';
+
 import { App as DoApp, AppConfig, AppSpecService, } from '@cdktf/provider-digitalocean/lib/app';
+import { TerraformOutput } from "cdktf";
+import { General } from "./general";
 
 
 export interface ServicesProps {
   config: PackageInfraConfig,
+  general: General,
   //vpc: Vpc,
 }
 
 export class Services extends Construct {
-  // readonly cluster: Cluster;
-  // readonly taskdefs: FargateTaskDefinition[];
-  // readonly services: FargateService[];
   private _app!: DoApp;
   public get app() {
```

Comment:
Missed this one last time, you could use `public readonly apps` as an alternative - a singular `app` may not hold for more than one service definitions?

Reply:
yes, you are right. This should create a new app instance for each instance of `detail`. Then `detail` should have a collection of services though could just stick to one in this API. I did look at refactoring here in this PR then decided against getting distracted.

Reply:
with that refactor the `get()` pattern will go. I know you prefer to stick with not using getter/setter, I haven't forgotten ;)